@here/harp-geoutils/lib/coordinates/GeoBox.ts

```diff
@@ -254,10 +258,19 @@ export class GeoBox implements GeoBoxExtentLike {
 
         const { west, east } = this;
 
-        if (east > west) {
-            return point.longitude >= west && point.longitude < east;
+        let longitude = point.longitude;
```

Comment:
is this tested somewhere?

Reply:
yes, I added a test for GeoBox.contains which implicitly tests this... see following test in GeoBoxTest: 
it("contains works for non normalized coordinates", function() {

Reply:
Sorry, I missed that.